Formula/r.rb

```diff
@@ -4,6 +4,7 @@ class R < Formula
   url "https://cran.r-project.org/src/base/R-3/R-3.6.3.tar.gz"
   sha256 "89302990d8e8add536e12125ec591d6951022cf8475861b3690bc8bf1cefaa8f"
   revision 1
+  head "https://stat.ethz.ch/R/daily/R-devel.tar.gz"
```

Comment:
This seems like an odd URL, HEAD should be a git repo.

Reply:
I found this link to their svn:
```suggestion
  head "https://svn.r-project.org/R/trunk/"
```